bodo/tests/test_iceberg/test_iceberg.py

```diff
@@ -4111,5 +4135,5 @@ def impl(table_name, conn, db_schema):
         )
         check_logger_msg(
             stream,
-            "Iceberg Filter Pushed Down:\nbic.FilterExpr('OR', [bic.FilterExpr('AND', [bic.FilterExpr('>', [bic.ColumnRef('A'), bic.Scalar(f0)]), bic.FilterExpr('NOT', [bic.FilterExpr('STARTS_WITH', [bic.ColumnRef('TY'), bic.Scalar(f1)])])]), bic.FilterExpr('IN', [bic.ColumnRef('B'), bic.Scalar(f2)])])",
+            "Iceberg Filter Pushed Down:\npie.Or(pie.And(pie.GreaterThan('A', literal(f0)), pie.Not(pie.StartsWith('TY', literal(f1)))), pie.In('B', literal(f2)))",
```

Comment:
Updated because we're using PyIcebergs filter classes instead of our own. `pie` stands for `pyiceberg.expressions`